report_pre_thesis/src/sections/background.tex

```diff
@@ -17,19 +8,22 @@ \section{Background}\label{sec:background}
 It allows for rapid analyses because of the immediate spectrum collection from the subsequent plasma, while maintaining a high spatial resolution due to its small observation footprints.
```

Comment:
```suggestion
It allows for rapid analyse because of the immediate spectrum collection from the subsequent plasma, while maintaining a high spatial resolution due to its small observation footprints.
```

Reply:
You can't suggest there because I didn't write this part:
https://github.com/chhoumann/thesis-chemcam/blob/f42ba9cd99d02eb29203846dc8f29a1f2b77293e/report_pre_thesis/src/sections/background.tex#L16-L17

Will fix it, though.

Reply:
And it shouldn't be `also`, from what I am reading here. The sentence refers to the previous mention of rapid analysis, so it's an explanation of why, not an addition.